.github/pull_request_template.md

```diff
@@ -0,0 +1,5 @@
+## Checklist

```

Comment:
Just a small suggestion, to add a placeholder reference to an issue that the PR is addressing:
```suggestion
Fixes #ISSUE

## Checklist
```